### DISCUSSION TITLE
希望可以支持多字段排序

### DISCUSSION THREAD

**Discussion by ideaviewes** [layui/layui#2268] [建议]
在统计表中，通常有多个字段同时排序的需求，把排序字段和排序方式已数组的形式传递到后台，由后台来完成排序，自己实现的话，非常麻烦，需要改源代码，希望官方能提供原生的支持。

**Comment by LSL1618** [comment]
多字段排序还是建议由后端接口中的SQL语句来实现，这是最优的。

**Reply by ideaviewes** [reply]
肯定是用后台接口实现的，现在的问题的是，layui的排序同时只能传一个排序的字段到接口

**Reply by LSL1618** [reply]
那就不要用表格来设置排序字段嘛，你把要排序的字段当成搜索条件，一个字段对应一个复选框或者一组单选框，这样不就行了。
